cli.js

```diff
@@ -0,0 +1,46 @@
+const readline = require('readline')
+const util = require('util')
+
+const Game = require('./lib/game.js')
+const Score = require('./lib/score.js')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const question = util.promisify(rl.question).bind(rl);
+const game = new Game()
+const score = new Score(game)
+
+console.log('WELCOME TO Bowling.JS')
+
+async function bowlingScoreCard() {
+  console.log('')
+  console.log(`-----> Score: ${score.total} <-----`)
+  console.log('')
+  try {
+    const answer1 = await question('Enter first roll: ')
+    const asnwer2 = await question('Enter second roll: ')
+    const answer3 = await question('Enter bonus roll: ')
```

Comment:
I find it interesting you've done the 3 rolls as separate answers - I just did mine with inputting one after the other! I can't quite tell, but does the bonus roll input still come up even if you don't have the bonus roll?